one_fm/operations/doctype/contracts/contracts.py

```diff
@@ -1042,7 +1042,7 @@ def send_contract_reminders(is_scheduled_event=True):
         users = [i.parent for i in relevant_users]
         if contracts_due_internal_notification:
             contracts_due_internal_notification_list = [[i.contract_termination_decision_period,i.contract_end_internal_notification,\
-                get_date_str(i.contract_termination_decision_period_date),i.name,get_date_str(i.start_date),get_date_str(i.contract_end_internal_notification_date),\
+                get_date_str(i.contract_termination_decision_period_date) if i.contract_termination_decision_period_date else None,i.name,get_date_str(i.start_date),get_date_str(i.contract_end_internal_notification_date) if i.contract_end_internal_notification_date else None,\
```

Comment:
Please, why are changes made to contracts here ?

I thought this was a PR for leave

Reply:
i didn't. You checked out from version-15